pkg/deployments/src/network.ts

```diff
@@ -39,3 +41,28 @@ export function getContractDeploymentTransactionHash(deployedAddress: string, ne
 
   return txHash;
 }
+
+export function saveContractDeploymentAddresses(task: Task): void {
+  if (task.network === 'hardhat') return;
```

Comment:
Do we need this? We'll pass an explicit `--network` flag

Reply:
I'm not sure. It's mostly there because we had it for `saveContractDeploymentTransactionHash` and we were building the lookup table in the same way as for deployment transactions initially. I'd say it doesn't hurt to keep it.